### ISSUE TITLE
Playback for files created by Capacitor.convertFileSrc

### ISSUE THREAD

**Issue by mcpeakdb** [jepiqueau/capacitor-video-player#48]
Hello! I'm having trouble with playback of file created by Capacitor.convertFileSrc() but only on iOS. The message I get back from the initPlayer is "filePath is not implemented". The path of these files on iOS begins with capacitor:// rather than http://, which I believe to be the root of the problem but was unable to figure out a workaround. 

Any suggestions or ideas for a fix? Or possibly a different way that I should create my webpath for this plugin?

Thanks for any help! Here is my code:

```
newpath =  /private/var/mobile/Containers/Data/Application/[app-id]/[file].MOV

await videoPlayer.initPlayer({
  url: Capacitor.convertFileSrc(newpath), // capacitor://localhost/_capacitor_file_/private/var/mobile/Containers/Data/Application/[app-id]/[file].MOV
  playerId: 'fullscreen', 
  mode: "fullscreen",
  componentTag: "div"
});
```

**Comment by jepiqueau** [comment]
@mcpeakdb Can you tell me were your file is located on the iOS device?. can you move the file to 
/data/Containers/Data/Applications/YOUR_APPLICATION/Documents/files/[file].mov 
in that case you will give `application/files/[file].mov 
i never use Capacitor.convertFileSrc()

**Comment by jepiqueau** [comment]
@mcpeakdb the alternative way has been fixed in 3.3.1-2 the iOS Plugin  accepts now
  . `file:///var/mobile/Media/DCIM/100APPLE/YOUR_VIDEO`
  . `file:///var/mobile/Containers/Data/Application/YOUR_APPLICATION_ID/tmp/YOUR_VIDEO`
 . `file:///var/mobile/Containers/Data/Application/YOUR_APPLICATION_ID/YOUR_VIDEO`

tell me if it is working for you
